tests/tst_ncio_mpi.F90

```diff
@@ -45,8 +45,8 @@ end subroutine check
 
   if (my_rank .eq. 0) print *,'*** Test that number of variables,dimensions,attributes is read...'
 
-  if (dsetin%nvars .ne. 24) stop 5
-  if (dsetin%ndims .ne. 6) stop 6
+  if (dsetin%nvars .ne. 25) stop 5
+  if (dsetin%ndims .ne. 7) stop 6
```

Comment:
Could you apply those constants to this file too? Might require separating it out into its own file.

Reply:
Done